_upgrading/sopel-8.md

```diff
@@ -0,0 +1,420 @@
+you _too much of a good thing_ and get stuck if the `logging_channel_level` was
+set to `DEBUG`.
+
+Therefore, `DEBUG` is no longer a valid choice for the `logging_channel_level`
+setting, and `logging_channel_level` is no longer inherited from the main
+`logging_level` setting. The new default `logging_channel_level` is `WARNING`.
+
+#### Farewell, Phenny/Jenni
+
+The project that eventually became Sopel diverged from Jenni in 2012. After more
+than a decade of maintaining backward compatibility (in a theoretical sense, at
+least) with plugins originally written for Jenni and its predecessor Phenny,
+Sopel 8.0 bids a fond farewell to plugin code from that era.
+
+Beyond the maintenance burden of making sure Sopel could still _technically_
+load such old plugins, we felt it was disingenuous to continue "supporting" them
+when the rest of Sopel's API has changed and evolved so much that the chance of
+such old plugins _actually still working_ is now very low indeed. Thus, we're
+not _dropping_ support for those decade-old plugins as much as we're _admitting_
+that _things have changed a lot_ and users would be better off seeking (or if
+necessary, writing) newer plugins with similar functionality.
+
+
+## Sopel 8 plugin changes
+
+### Removed built-in plugins
+
+Sopel's built-in plugins are slowly being migrated out to their own standalone
+packages, which will help the project manage responsibility for maintenance and
+updates in the long term. In 8.0, we say farewell to:
+
+* `help`: now published as [`sopel-help`][sopel-help]
+  * Note: Sopel still requires the `sopel-help` package, so it will be installed
+    and available automatically.
+* `ip`: now published as [`sopel-iplookup`][sopel-iplookup]
+* `meetbot`: now published as [`sopel-meetbot`][sopel-meetbot]
+* `py`: now published as [`sopel-py`][sopel-py]
+* `reddit`: now published as [`sopel-reddit`][sopel-reddit]
+* `remind`: now published as [`sopel-remind`][sopel-remind]
+
+  [sopel-help]: https://pypi.org/project/sopel-help/
+  [sopel-iplookup]: https://pypi.org/project/sopel-iplookup/
+  [sopel-meetbot]: https://pypi.org/project/sopel-meetbot/
+  [sopel-py]: https://pypi.org/project/sopel-py/
+  [sopel-reddit]: https://pypi.org/project/sopel-reddit/
+  [sopel-remind]: https://pypi.org/project/sopel-remind/
+
+### Notable changes to built-in plugins
+
+#### `currency` plugin
+
+The `fiat_provider` setting now takes precedence over the `fixer_io_key`.
+Previously, setting a `fixer_io_key` would use the `fixer.io` fiat exchange rate
+provider regardless of the `fiat_provider` setting.
+
+The old `fiat_provider` default of `exchangerate.host` is no longer a valid
+choice; it has been replaced by `open.er-api.com`.
+
+#### `reload` plugin
+
+The `.update` command has been removed from the `reload` plugin. Its functioning
+relied on running Sopel in a way that isn't officially supported, so we chose to
+remove it.
```

Comment:
We should probably point users who used to depend on `.update` towards `.reload` or restarting Sopel. Guessing that the latter is the preferred option.

Reply:
This is a good idea; I ran with it.

I'm reasonably sure about the URLs, but not 100% certain as the Netlify configuration could differ slightly between the `unstable.docs` for 8.0.0 and the main site.

```suggestion
The `.update` command has been removed from the `reload` plugin. It relied on
running Sopel in a way that we don't officially support, so we chose to remove
the command.

To replace `.update`, if you were using it, we recommend writing an external
script that [stops Sopel][cli-stop], updates the components you want, and then
[starts Sopel][cli-start] again.

  [cli-stop]: https://sopel.chat/docs/run/cli.html#sopel-stop
  [cli-start]: https://sopel.chat/docs/run/cli.html#sopel-start
```

Reply:
(We can avoid the uncertainty about how the final URLs will look by skipping the links for now, mentioning `sopel start` and `sopel stop` commands directly, and linkifying them post-release if desired.)

Reply:
The `.reload`  should never ever be used or promoted. We collectively spent too much time on trying to figure out a way to make it work and it still doesn't.

Reply:
@Exirel Are you saying not to mention the removal of `.update` at all? I'm not sure what change (if any) you are indirectly suggesting to the proposed migration guide.

Reply:
It's a reply to SnoopJ's initial comment to talk about `.reload`.